interfaces/udev/spec.go

```diff
@@ -41,7 +41,8 @@ type Specification struct {
 	entries  []entry
 	iface    string
 
-	securityTags []string
+	securityTags            []string
+	udevadmSubsystemTrigger string
```

Comment:
As indicated above, perhaps this ought to be a set (or a simple list perhaps)

Reply:
Done